snippets/python/Diamond_Square/Diamond_Square.md

```diff
@@ -0,0 +1,37 @@
+# Diamond Square
+
+## Description
+A Program to print the Diamond Sqaure Pattern. The rows count is given as a input which denotes the longest row in the pattern.
+
+## Code
+```python
+# The Diamond Pattern
+n = int(input("Enter the rows count:"))
+
+# The Top Pyramid
+for i in range(n):
+    space = "  " * (n - i - 1)
+    star = (2 * i) * "* " + "*"
+    print(space + star)
+
+# The bottom Pyramid
+for i in range(n):
+    space = "  " * (i)
+    star = star[:-4]
+    print("  " + space + star)
+```
+
+## Input
+```
+Enter the rows count: 7
+```
+
```

Comment:
The input line will already be visible in the output image. So there's no need for an extra sub heading.
```suggestion
```